tardis/montecarlo/base.py

```diff
@@ -302,11 +320,40 @@ def to_hdf(self, path_or_buf, path=''):
         """
         runner_path = os.path.join(path, 'runner')
         properties = ['output_nu', 'output_energy', 'nu_bar_estimator',
-                      'j_estimator', 'montecarlo_virtual_luminosity']
+                      'j_estimator', 'montecarlo_virtual_luminosity',
+                      'last_line_interaction_in_id',
+                      'last_line_interaction_out_id',
+                      'last_line_interaction_shell_id',
+                      'packet_luminosity', 'output_nu'
+                      ]
         to_hdf(path_or_buf, runner_path, {name: getattr(self, name) for name
                                           in properties})
         self.spectrum.to_hdf(path_or_buf, runner_path)
         self.spectrum_virtual.to_hdf(path_or_buf, runner_path,
                                      'spectrum_virtual')
         self.spectrum_reabsorbed.to_hdf(path_or_buf, runner_path,
                                         'spectrum_reabsorbed')
+
+    @classmethod
+    def from_config(cls, config):
+        if config.plasma.disable_electron_scattering:
+            logger.warn('Disabling electron scattering - this is not physical')
+            sigma_thomson = 1e-200 / (u.cm ** 2)
```

Comment:
After some digging in the code, it seems that the unit of sigma_thomson is completely ignored afterwards, since cmontecarlo uses just its value. So, neither the wrong unit, neither correcting it would affect the computations in any way, which means it is probably safe to fix this by directly using, as @chvogl proposed, the `astropy.constants.sigma_T`.